README.md

```diff
@@ -0,0 +1,376 @@
+              "name": "SummaryText",
+              "schema": "string"
+            },
+            {
+              "name": "highTemp",
+              "schema": "double"
+            },
+            {
+              "name": "lowTemp",
+              "schema": "double"
+            },
+            {
+              "name": "avgTemp",
+              "schema": "double"
+            },
+            {
+              "name": "numberOfSamples",
+              "schema": "float"
+            }
+          ]
+        }
+      }
+    }
+  ]  
+}
+```
+
+</details>
+
+The WeatherStation device implement the next interfaces:
+
+<details>
+
+<summary>WeatherStation model</summary>
+
+```json
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azsamples:iotdd:WeatherStation;1",
+  "@type": "Interface",
+  "displayName": "Sample Weather Station with interior and exterior sensors",
+  "contents": [
+    {
+      "@type": "Component",
+      "schema": "dtmi:azsamples:iotdd:ClimateSensor;1",
+      "name": "interior"
+    },
+    {
+      "@type": "Component",
+      "schema": "dtmi:azsamples:iotdd:ClimateSensor;1",
+      "name": "exterior"
+    },
+    {
+      "@type": "Component",
+      "schema": "dtmi:azure:DeviceManagement:DeviceInformation;1",
+      "name": "deviceInfo"
+    },
+    {
+      "@type": "Component",
+      "schema": "dtmi:azure:Client:SDKInformation;1",
+      "name": "sdkInfo"
+    },
+    {
+      "@type": "Component",
+      "schema": "dtmi:azsamples:iotdd:Diagnostics;1",
+      "name": "diag"
+    }
+  ]
+}
+```
+
+</details>
+
+### CloudAplication
+
+This is an application that connects to IoT Hub using the service SDK offering the next features:
+
+- List Devices and Status
+- For Each Device:
+  - Determine if the device announces a ModelId
+  - Resolve the Model contents from the ModelId
+    - If it's a known model, show an ad-hoc UI
+  - For Models without an ad-hov UI
+    - Show a dynamically generated UI
+  - Devices without ModelId
+    - Show the properties available in the Twin
+
+## SDK support
+
+This release is supported by the next SDK versions
+
+Device SDK
+
+- azure-iot-sdk-c [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp) branch
+- azure-iot-sdk-node [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-node/tree/public-preview-pnp) branch
+- azure-iot-sdk-python [digitaltwins-preview](https://github.com/Azure/azure-iot-sdk-python/tree/digitaltwins-preview) branch
+
+Service SDK
+
+- azure-iot-sdk-node [public-preview-pnp](https://github.com/Azure/azure-iot-sdk-node/tree/public-preview-pnp) branch 
+
+## Scenario Steps
+
+### 1. Create the Thermostat Device simulator
+
+Based on the sample device for [C](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp/digitaltwin_client/samples) or [node](https://github.com/Azure/azure-iot-sdk-node/tree/public-preview-pnp/digitaltwins/samples/device), adapt the sample code to implement the Thermostat model.
```

Comment:
I think these steps are good for a preview of the bug bash, given we don't have the documentations now. In the bug bash, I'd prefer to give a link to the article instead, where it should include things like where to get samples, what the happy path etc. Things to try are good examples for people to bug bash more.

Reply:
Agree, I will add links to articles using the staging docs.

Reply:
I think we (SDK Team) should provide an happy path for Thermostat (providing an implementation) and invite the bugbasher to implement the  weather station.